text/0000-fix-daa.md

```diff
@@ -0,0 +1,113 @@
+and responsiveness. They may also achieve greater overall simplicity, both conecptually and in implementation.
+
+# Prior art
+[prior-art]: #prior-art
+
+The current DAA in Bitcoin Cash suffers from the same oscillatory behaviour that Grin's DAA does.
+Which is what led various BCH developers to do extensive research on possible alternatives [1] [2].
+They identified two DAAs as being superior to others. These two, wtema, and asert, also happen to behave nearly identically.
+While wtema is simpler in that it doesn't need exponentiation, it would need special safeguards to robustly deal with
+the possibility of very negative solvetimes. Since Grin requires strictly increasing timestamps, it doesn't need any such
+safeguards, making wtema the preferred choice for Grin.
+
+Before adding a dependence on number of parent uncles, Ethereud uses a DAA that can be seen as a close approximation of wtema:
+After every block, increase network difficulty by a factor of 1 - (t/T - 1) / M.
+Comparing with wtema above, we see that instead of dividing by (1+x), they multiply by (1-x).
+Even though Ethereum, like Grin, enforces positive solvetimes, they still need to guard against division by 0 in this form.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+- What parts of the design do you expect to resolve through the RFC process before this gets merged?
+While this RFC argues for adopting wtema as a new DAA, it doesn't want to argue for a specific value of the half life M.
+In Grin tradition, an integral number of hours is preferable, to limit
+arbitrariness. Either 2 hours, 3 hours, or 4 hours, all offer a resonable
+balance of responsiveness and stability.
+These 3 choices as well as Grin's current DAA with various damping factor choices are implemented at [3],
+which allows for easy comparison across many scenarios.
+At the same time, we need to decide if reducing the maximum allowed clock drift (FTL parameter) from 12 minutes
+down to 1 makes sense, as this will reduce one form of timestamp manipulation.
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+Although Grin graphrate has been relatively stable throughout its short history, as a GPU mineable coin availble at nicehash, it's always at risk of sudden temporary surges in graphrate, to which it should be able to respond quickly. The current DAA does that, but at a cost (of oscillatory behaviour).
+In a future where Grin mining is dominated by Cuckatoo32 ASICs and where Grin is the C32 coin with the largest daily issuance, 
+the graphrate should be naturally stable, with the need for responsiveness lessened.
+We should however account for the possibility of other coins adopting Cuckatoo32 as well.
+If, like most coins, they opt for a finite supply, then in initial years they may well have a daily issuance exceeding that of Grin. That would put us in a situation similar to Bitcoin Cash competing with Bitcoin for ASIC hashrate.
```

Comment:
Does it put _us_ in the bitcoin cash position or the other coin in the bitcoin cash position?
This is not clear as written.